### PR TITLE
Fix Tryout page accessibility issues

### PR DESCRIPTION
**Pull request by erangatl**
Fixed issues 1 to 3 https://github.com/wso2/product-apim/issues/8379

image

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx [hunk 1]
```diff
@@ -565,6 +565,7 @@ function TryOutController(props) {
                                             defaultMessage='Enter access Token'
                                         />
                                     )}
+                                    id='accessTokenInput'
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:568]: why we need an ID here ?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx [hunk 2]
```diff
@@ -619,7 +621,12 @@ function TryOutController(props) {
                                             />
                                         )}
                                     >
-                                        <HelpOutline className={classes.tooltip} />
+                                        <IconButton
+                                            aria-label='Use existing Access Token or generate a new Test Key'
+                                            style={{ margin: 8, height: '48px' }}
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:626]: We could wrap this in a <Box /> applying these margin and height styles

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/SelectAppPanel.jsx [hunk 3]
```diff
@@ -57,7 +57,7 @@ const SelectAppPanel = (props) => {
             <Grid x={12} md={6} className={classes.centerItems}>
                 <TextField
                     fullWidth
-                    id='outlined-select-currency'
+                    id='selected-application'
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/SelectAppPanel.jsx:60]: I think we don't need IDs here and below as well , can remove them

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx (hunk 1 above)
Comment (erangatl) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx:568]: When we add the ID it generates the ` aria-describedby ` for accessibility

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/SelectAppPanel.jsx (hunk 3 above)
Comment (erangatl) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/SelectAppPanel.jsx:60]: When we add the ID it generates the ` aria-describedby ` for accessibility

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx (hunk 2 above)
Comment (erangatl) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx]: Fixed

**Pull request merged**